Changes in risk of sex offender section

lostie opened the pull request:
[Trello #370](https://trello.com/c/FbTwsENl/370-1-as-someone-working-in-security-in-a-prison-when-filling-in-a-digital-per-i-want-to-be-able-to-complete-details-about-a-detaine)

- Whenever the user select that the prisoner is a sex offender, a type
of victim needs to be selected. These changes enforce that happens.
- Minor wording changes
- **Due to the new design, the sex offences victim types changes from radio to checkboxes, hence the extra changes required.**

Commit:
Add validation check for victim type if sex offence is set to 'Yes'

- Whenever the user select that the prisoner is a sex offender, a type
of victim needs to be selected. These changes enforce that happens.
- Minor wording changes

dan-palmer commented on app/models/forms/risk/sex_offences.rb:
```diff
@@ -17,8 +17,7 @@ class SexOffences < Forms::Base
       validates :sex_offence_victim,
-        inclusion: { in: VICTIM_VALUES },
-        allow_blank: true
+        inclusion: { in: VICTIM_VALUES }, if: proc { |f| f.sex_offence == 'yes' }
```
This could be written like:

```ruby
inclusion: { in: VICTIM_VALUES }, if: -> { sex_offence == 'yes' }
```

Commit:
Change sex offender victim from radio to checkboxes

Due to the nature of this changes the following was required:
- Remove old columns that are no longer necessary for the sex offences
questions
- Add new columns necessary for the sex offences question
- Remove custom presenter for sex offences summary (yay!)

lostie commented:
@dan-palmer This needs reviewing (again), as there was an extra change that wasn't really clear in the story, but was in the design changes. The radios are now checkboxes :S

dan-palmer commented on app/models/forms/risk/sex_offences.rb:
```diff
@@ -4,27 +4,37 @@ class SexOffences < Forms::Base
 
-      property :sex_offence_victim, type: StrictString
+      def valid_sex_offence_options
+        unless selected_sex_offence_options.any?
```
Could be: `if selected_sex_offence_options.none?`
